packages/nextjs/components/scaffold-eth/Contract/utilsComponents.tsx

```diff
@@ -83,14 +83,14 @@ const TxValueInput = ({ setTxValue, txValue }: { setTxValue: Dispatch<SetStateAc
           data-tip="Multiply by 10^18 (wei)"
         >
           <button
-            className="cursor-pointer text-xl font-semibold pt-1 px-4 text-accent"
+            className="cursor-pointer text-l font-semibold px-4 text-accent"
```

Comment:
just small changes, we can change `text-l` -> `text-xl`.....I think it was added by mistake since `text-l` doesn't exist in tailwind

Also I think we should keep it `text-xl` instead of removing it I like big symbols

Reply:
Oops, wanted to put `text-lg`. I modified it to keep the sizing of the asterisk about the same, but if everyone is fine with a larger character, we can keep it on text-xl as well. Either works for me.

Reply:
cc @carletex for `xl` and `lg`... both look great to me, maybe he can change it while merging